dotnet/Capstone/DAO/BirdSqlDao.cs

```diff
@@ -12,21 +13,54 @@ namespace Capstone.DAO
     public class BirdSqlDao : BirdDao
     {
         private readonly string connectionString;
+        private readonly IBirdListDao birdListDao;
        
 
         public BirdSqlDao(string dbConnectionString)
         {
             connectionString = dbConnectionString;
+            this.birdListDao = new BirdListSqlDao(connectionString);
             
         }
 
+        public List<Bird> getBirdsInList(int listId)
+        {
+            List<Bird> birdList = new List<Bird>();
 
-        public Bird createBird(Bird bird, string name, string description, string picture)
+            string sql = "SELECT id, list_id, name, picture, zip_code FROM bird WHERE list_id = @list_id";
+
+            try
+            {
+                using (SqlConnection conn = new SqlConnection(connectionString))
+                {
+                    conn.Open();
+
+                    SqlCommand cmd = new SqlCommand(sql, conn);
+                    cmd.Parameters.AddWithValue("@list_id", listId);
+
+                    SqlDataReader reader = cmd.ExecuteReader();
+                    
+                    while (reader.Read())
+                    {
+                        Bird bird = MapRowToBird(reader);
+                        birdList.Add(bird);
+                    }
+                }
+            }
+            catch (SqlException ex)
+            {
+                throw new DaoException("SQL exception occurred", ex);
+            }
+
+            return birdList;
+        }
+
+        public Bird createBird(Bird bird, int listId)
         {
             Bird newBird = null;
 
-            String sql = "INSERT INTO bird(name, description, picture) " +
-                "VALUES(@name, @description, @picture)";
+            String sql = "INSERT INTO bird(list_id, name, picture, zip_code) " +
+                "VALUES(@list_id, @name, @picture, @zip_code)";
 
             int newBirdId = 0;
```

Comment:
newBirdId is assigned 0 but isn't used. Did you intend to return this number from your SQL statement? If not, it doesn't seem to be serving any purpose.

Reply:
This is just declaring the new bird id which will be later set to the newly created bird object.